### PR TITLE
[WIP] scripts,lib: add node-land-checks

### PR DESCRIPTION
**Pull request by maclover7**
This is what a `github-bot` assisted version of https://github.com/nodejs/node/pull/22275 could look like.

cc @phillipj @Trott @jasnell

### CODE REVIEW

File: lib/node-land-checks.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const url = require('url')
+
+const githubClient = require('../lib/github-client')
+
+const threeDaysAgo = (24 * 60 * 60 * 1000) * 3
```
[the remark this answers is not in the document]
Comment (joyeecheung) [reply lib/node-land-checks.js]: Isn't it 48 hours for UTC weekdays and 72 hours for UTC weekends?

[the remark this answers is not in the document]
Comment (Trott) [reply lib/node-land-checks.js]: @joyeecheung Yes, but I think @maclover7 is offering this as a something we can use if https://github.com/nodejs/node/pull/22275 lands. By the way, if you want to weigh in on that PR, that would be great.

**Comment by phillipj** [comment]
@maclover7 should this still be considered for merging and production rollout, or closed for now?

**Comment by phillipj** [comment]
Closing this for now due to lack of activity.

**Pull request closed without merging**